sdk/spring/spring-cloud-azure-autoconfigure/pom.xml

```diff
@@ -300,12 +300,6 @@
       <version>3.0.0-RC1</version> <!-- {x-version-update;org.springframework.boot:spring-boot-configuration-metadata;external_dependency} -->
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>org.glassfish</groupId>
-      <artifactId>jakarta.el</artifactId>
-      <version>4.0.2</version> <!-- {x-version-update;org.glassfish:jakarta.el;external_dependency} -->
-      <scope>test</scope>
-    </dependency>
```

Comment:
Why is this?

Reply:
It will be not managed by Spring Boot RC2, and we didn't use any classes from this dependency in our test code.